util/src/test/java/io/zeebe/util/logging/RecordingAppender.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.0. You may not use this file
+ * except in compliance with the Zeebe Community License 1.0.
+ */
+package io.zeebe.util.logging;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+import org.apache.logging.log4j.core.Appender;
+import org.apache.logging.log4j.core.ErrorHandler;
+import org.apache.logging.log4j.core.Layout;
+import org.apache.logging.log4j.core.LogEvent;
+
+public final class RecordingAppender implements Appender {
```

Comment:
This class seems to really be a testing utility class. I'm not sure this is the best package for it. I would at least expect some documentation on this class.

Reply:
Which package would you put it in?

Reply:
Oh sorry, it is already in the test sources folder. My mistake, the package is fine. I would appreciate some use case or behaviour description as the javadoc on this class though